### PR TITLE
Add LTS 202406.00 release 

### PR DESCRIPTION
**Pull request by chinglee-iot**
**Description of changes:**
* Add LTS 202406.00 release 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by kar-rahul-aws** [comment]
CODEOWNERS file can be added to require PR-BR review before merging of a pull request.

File: README.md [hunk 1]
```diff
@@ -5,25 +5,42 @@ AWS also offers FreeRTOS Extended Maintenance Plan (EMP) that provides you with
...
-| coreJSON                    | 3.0.0               | 03/31/2023 | https://github.com/FreeRTOS/coreJSON/tree/v3.0.0                            |
-| backoffAlgorithm            | 1.0.0               | 03/31/2023 | https://github.com/FreeRTOS/backoffAlgorithm/tree/v1.0.0                    |
-| AWS IoT Device Shadow       | 1.0.2               | 03/31/2023 | https://github.com/aws/Device-Shadow-for-AWS-IoT-embedded-sdk/tree/v1.0.2   |
-| AWS IoT Device Defender     | 1.1.0               | 03/31/2023 | https://github.com/aws/Device-Defender-for-AWS-IoT-embedded-sdk/tree/v1.1.0 |
-| AWS IoT Jobs                | 1.1.0               | 03/31/2023 | https://github.com/aws/Jobs-for-AWS-IoT-embedded-sdk/tree/v1.1.0            |
-| AWS IoT Over-the-air Update | 3.0.0               | 03/31/2023 | https://github.com/aws/ota-for-aws-iot-embedded-sdk/tree/v3.0.0             |
+Libraries in this GitHub branch (also listed below) are part of the [FreeRTOS 202210-LTS](https://github.com/FreeRTOS/FreeRTOS-LTS/tree/202210-LTS) release. Learn more at https://freertos.org/lts-libraries.html.
+
```
Comment (sirnish) [inline README.md:9]: FreeRTOS 202210-LTS](https://github.com/FreeRTOS/FreeRTOS-LTS/tree/202210-LTS should be probably 202406

Comment (chinglee-iot) [reply README.md:9]: Update the version and remove the link here. Thanks for the catch.

**Comment by chinglee-iot** [comment]
> CODEOWNERS file can be added to require PR-BR review before merging of a pull request.

Thank you. Added in new commit.

**Review by aggarg — APPROVED** [review]

**Review by ActoryOu — APPROVED** [review]

**Pull request merged**